pages/1.12/release-notes/1.12.3/index.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.12.3
+title: Release Notes for 1.12.3
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.12.3
+---
+
+DC/OS Version 1.12.3 was released on March 14, 2019.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/1.12.3/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise [/button]
+
+DC/OS 1.12.3 includes the following components:
+- Apache Mesos 1.7.x [change log](https://github.com/apache/mesos/blob/1.7.x/CHANGELOG).
```

Comment:
I would put the link to the SHA instead of the branch for consistency
https://github.com/apache/mesos/blob/5e234c8d8edc1bb73ba557f5774c609fa460c9e7/CHANGELOG